src/service/resources-impl.js

```diff
@@ -1628,16 +1655,15 @@ export class Resource {
   /**
    * Requests the resource's element to be built. See {@link AmpElement.build}
    * for details.
-   * @param {boolean} force
    * @return {boolean}
```

Comment:
hah. I've kinda removed it from here, should I bring it back? I tried to see where do we actually use it, couldn't find a place where we do check the returned value, but in the future, it might be useful.